tests/unit/specs/service/auth.spec.js

```diff
@@ -133,7 +134,7 @@ describe('auth service', () => {
       const res = await authService.getAuthStatus();
 
       expect(res).toEqual({
-        status: 200,
+        status: AUTH_STATUS_CODE.LOGGED_IN,
```

Comment:
maybe can we rename AUTH_STATUS_CODE to AUTH_STATUS?

Reply:
it will crossed with `METHODS.AUTH_STATUS` that's why i named as `AUTH_STATUS_CODE` =\